src/frontend/src/components/common/Navbar/index.tsx

```diff
@@ -5,23 +5,25 @@ import { INavbarPages } from './interfaces';
 function NavBar({ pages }: INavbarPages): ReactElement {
   return (
     <nav className="navbar-container ">
-      <ul className="navbar ms-auto">
-        {pages.map((page) => {
-          return page.link === '/' ? (
+      {pages.map((page) => {
+        return page.link === '/' ? (
+          <ul className="navbar">
             <li key={page.id}>
               <NavLink to={page.link} end>
                 {page.label}
               </NavLink>
             </li>
-          ) : (
+          </ul>
+        ) : (
+          <ul className={page.id % 2 === 0 ? ' navbar ms-auto' : 'navbar'}>
```

Comment:
Is this only applicable to even pages? What does 'even' pages mean? I would avoid this type of check. adding a class 'ms-auto' to this check is not very clear.

Reply:
![Menu](https://user-images.githubusercontent.com/57013495/203617911-38a2c686-0ef5-4d1e-a9d9-cb23ebe08fc0.png)
it follows the design, make sure one in left and another one in right for navbar item

Reply:
There is a better way to do this. I would avoid this type of logic. 
Please have one class and make sure it fits both designs. With one button or two button and it should render correctly.

Reply:
just remove this